examples/drawings.js

```diff
@@ -32,6 +32,24 @@ const regionToColor = region => {
   }
 };
 
+const selectedCategory = 'teapot';
+const categories = [
+  'flower',
+  'piano',
+  'power outlet',
```

Comment:
Please make all names lower case and replace white spaces with dashes. Thanks
```suggestion
  'power-outlet',
```

Reply:
Remember to rename the json file as well! 😄